src/core/mainwindow.cpp

```diff
@@ -73,6 +73,8 @@
 #include <QToolButton>
 #include <QCheckBox>
 #include <QClipboard>
+#include <QDBusConnection>
```

Comment:
You need to use `#ifdef HAVE_DBUS` around all this code including the function.

Reply:
Thanks, that's a good call.  I'll make the changes and update the PR.

Reply:
Done